src/components/Gradebook/index.jsx

```diff
@@ -259,6 +252,11 @@ export default class Gradebook extends React.Component {
                 The grades for this course are now frozen. Editing of grades is no longer allowed.
               </div>
             }
+            { !this.props.canUserViewGradebook &&
+              <div className="alert alert-warning" role="alert" >
+                You are not authorized to view the gradebook for this course. If you have a global role, try enrolling in this course and try again.
```

Comment:
The word `try` is used twice. Please only use that word once, like `If you have a global role, please enroll into this course and try again`